app/models/analyzer.rb

```diff
@@ -0,0 +1,85 @@
+class Analyzer
+  INCREMENT_NUM_PER_TEXT = 100
+
+  def initialize(use_persistent: false)
+    @uri = URI.parse("#{Rails.configuration.elasticsearch[:host]}/entries/_analyze")
+    @http =
+      if use_persistent
+        Net::HTTP::Persistent.new
+      else
+        Net::HTTP.new(@uri.host, @uri.port)
+      end
+    @post = Net::HTTP::Post.new(@uri.request_uri, 'Content-Type' => 'application/json')
+    @use_persistent = use_persistent
+  end
+
+  def normalize(text, normalizer)
+    raise ArgumentError, "Empty text" unless text.present?
+    _text = text.tr('{}', '()')
+    body = { analyzer: normalizer, text: _text }.to_json
+    response = tokenize(body)
+
+    (JSON.parse response.body, symbolize_names: true)[:tokens].map{ _1[:token] }.join('')
```

Comment:
```suggestion
    JSON.parse(response.body, symbolize_names: true)[:tokens].map{ _1[:token] }.join('')
```
カッコに位置が見慣れないスタイルに思えます。

Reply:
ご指摘ありがとうございます。
co-commitさせていただきました。
https://github.com/pubannotation/pubdictionaries/pull/157/commits/16101edbc93612d0a78210f3d5fb742a1525861b